### ISSUE TITLE
evented vs. action confusion

### ISSUE THREAD

**Issue by jayphelps** [trek/ember-evented-checkbox#1]
This is really a problem with Ember, which trickles into this project. `Ember.Evented` creates it's own concept of an "event", unrelated to `window.Event`, which itself is confusion right off the bat. On top of this, actions aren't related to `Ember.Evented` in any way, except that you may use one to trigger the other. This again, is confusing.

How this related to this project is that it's unclear what an "evented-checkbox" is. It uses `Ember.Evented` internally, but this is more of an implementation detail rather than the merit, which seem s to me that it emits an Ember action on change. While we can't solve Ember's naming conventions here, I wonder if this project and components could be named more accurately; something like `<actionable-checkbox>` or something.

This may seem like bikeshedding (OK...may it is)..but this project is an important testing ground for future Ember IMO and the sooner we start reconciling action vs. ember event vs. window.Event the better.

**Comment by jayphelps** [comment]
For lurkers who are unaware:

| Concept               | Emitting                                             | Bubbles                      | Cancelable                      |
|-----------------------|------------------------------------------------------|------------------------------|---------------------------------|
| `Ember.ActionHandler` | `{{action}}`, `this.send(name)`, `this.sendAction()` | Yes (via `.target` property) | Yes (don't `return true`)       |
| `Ember.Evented`       | `this.trigger(name, args)`                           | No                           | n/a                             |
| `window.Event`        | `el.dispatchEvent(new Event(name));`                 | Yes (via `.parentNode`)      | Yes (`event.stopPropagation()`) |

**Comment by trek** [comment]
You're right, the name isn't correct. "actionable" is awful though. There's no way I'd go with that. "event" copies from the React style of naming for passing callback/reference to a callback, which is why it was selected.

**Comment by jayphelps** [comment]
![¯\_(ツ)_/¯](http://gifrific.com/wp-content/uploads/2013/03/Joey-Tribianni-Shrug-Friends.gif)

**Comment by trek** [comment]
I think, more likely, is we need to remove some of those other concepts for 2.0
